src/cdk/v2/destinations/koala/procWorkflow.yaml

```diff
@@ -0,0 +1,28 @@
+bindings:
+  - name: EventType
+    path: ../../../../constants
+  - name: defaultRequestConfig
+    path: ../../../../v0/util
+
+steps:
+  - name: validateInput
+    template: |
+      $.assert(.message.type, "message Type is not present. Aborting message");
+      $.assert(.message.type in {{$.EventType.([.IDENTIFY, .TRACK])}}, 
+        "message type " + .message.type + " is not supported");
+      $.assertConfig(.destination.Config.publicKey, "publicKey is not present. Aborting message");
+      $.assert(.message.traits.email, "email is not present on traits. Aborting message");
```

Comment:
Is it possible to check for either `email` OR `profile_id` – if they set the Koala profile id in the traits we should be able to use that too.

We may want to have a more specific name than `profile_id` though, something to indicate it's the Koala id: `ko_profile_id`?